build.gradle

```diff
@@ -113,7 +113,7 @@ allprojects {
       '-Xlint:divzero',
       '-Xlint:finally',
       '-Xlint:static',
-      '-Werror',
+//      '-Werror',
```

Comment:
This will allow warnings anywhere in Artemis which we've wanted to avoid. We probably just have to add `@SuppressWarnings` for the 21 or so warnings in the new discovery module.

Reply:
done